README.md

```diff
@@ -26,6 +26,35 @@ The plugin is available from the [Wordpress plugin repo][11].
 3. Activate the plugin through the Plugins menu in WordPress admin.
 4. Set the `Shop Account ID` to the plugin's settings.
 
+## Filters
+
+The plugin provides [filters][12] that allows you to customize some fields that will be reported to analytics in an order. The filters
+in the plugin are as follows:
+
+1. The filter tag is: `wc_skroutz_analytics_product_id_filter`, the value is the generated
+   Unique ID and the additional argument is the product object (WC_Product | WC_Product_Variable).
+
+   Usage example:
+
+    ```
+    function my_product_id( $id, $product ) {
+        return "my-{$id}-custom-{$product->get_sku()}";
+    }
+    add_action( 'wc_skroutz_analytics_product_id_filter', 'my_product_id', 10, 2 );
+    ```
+
+2. The filter tag is: `wc_skroutz_analytics_product_reviews_widget_id`, the value is the generated
+   Unique ID with the term ID and the additional argument is the product object (WC_Product | WC_Product_Variable).
+
+   Usage example:
+
+    ```
+    function my_product_variation_id( $variation_id, $product ) {
```

Comment:
[minor] I would rename the function name as well to sth like: "my_product_reviews_widget_id". Don't forget the change the callback name in the add_action as well.